<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JPEGClass</class>
 <widget class="QWidget" name="JPEGClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JPEG编解码器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../BaiduNetdiskDownload/原神icon/icon/10388.ico</normaloff>../../../BaiduNetdiskDownload/原神icon/icon/10388.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="openvideo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>打开视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getflash">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>提取关键帧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JPEG_B">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>编码.jpg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JPEG_J">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>解码.jpg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JPEG_B_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>编码.dat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JPEG_J_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>解码.dat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>640</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="JPEG.qrc">:/JPEG/taozi2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>425</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="JPEG_B_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>质量因子</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="value">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>openvideo</zorder>
  <zorder>getflash</zorder>
  <zorder>JPEG_B</zorder>
  <zorder>JPEG_J</zorder>
  <zorder>JPEG_B_2</zorder>
  <zorder>JPEG_J_2</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>JPEG_B_3</zorder>
  <zorder>value</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="JPEG.qrc"/>
 </resources>
 <connections/>
</ui>
